*** ./node_modules/mocha/lib/reporters/templates/coverage.jade	Wed Jan  2 19:49:05 2013
--- ./node_modules/mocha/lib/reporters/templates/coverage.jade	Wed Jan  2 19:49:21 2013
***************
*** 37,50 ****
                      tr.hit 
                        td.line= number
                        td.hits= line.coverage
!                       td.source= line.source
                    else if 0 === line.coverage
                      tr.miss 
                        td.line= number
                        td.hits 0
!                       td.source= line.source
                    else
                      tr
                        td.line= number
                        td.hits
!                       td.source= line.source || ' '
\ No newline at end of file
--- 37,50 ----
                      tr.hit 
                        td.line= number
                        td.hits= line.coverage
!                       td.source!= line.source
                    else if 0 === line.coverage
                      tr.miss 
                        td.line= number
                        td.hits 0
!                       td.source!= line.source
                    else
                      tr
                        td.line= number
                        td.hits
!                       td.source!= line.source || ' '
\ No newline at end of file
